core/common/src/IpcAppProps.ts

```diff
@@ -73,8 +83,10 @@ export interface IpcAppFunctions {
   /** Determine whether there are outstanding txns . */
   hasPendingTxns: (key: string) => Promise<boolean>;
 
-  pullAndMergeChanges: (key: string) => Promise<IModelConnectionProps>;
-  pushChanges: (key: string, description: string) => Promise<IModelConnectionProps>;
+  /** pull (and potentially merge local) changesets up to the specified version into this Briefcase. */
+  pullAndMergeChanges: (key: string, version?: IModelVersionProps) => Promise<void>;
+
+  pushChanges: (key: string, description: string) => Promise<string>;

```

Comment:
What is the meaning of the return value?

Reply:
Apparently it's the changesetId.

Reply:
see the "real" API at `BriefcaseConnection.pushChanges`. Really the @internal interface definition is not the place to document usage. It is misleading to have extensive documentation here since it's not intended for anyone to call directly (though sometimes the interface methods are only used internally, and then have some internal-only documentation).

Reply:
added notes to see real api